### PR TITLE
Create a battle

### PR DESCRIPTION
**Pull request by pantoja**
**How to test:**

1. While the db is still in the project, login with `admin@admin.com` and password `senhaadmin`
2. Access http://127.0.0.1:8000/create/ and create a battle choosing an opponent.

### CODE REVIEW

File: .prospector.yaml [hunk 1]
```diff
@@ -33,6 +33,8 @@ pylint:
     - invalid-name
     - no-self-use
     - no-member
+    - too-many-ancestors
+    - import-error  #TODO: Remove this exception
```
[the remark this answers is not in the document]
Comment (pantoja) [reply .prospector.yaml]: Could you explain what "tracked" means?

File: backend/battles/views.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+from battles.forms import BattleForm
+from battles.models import Battle
+
+
+class CreateBattleView(LoginRequiredMixin, CreateView):
+    template_name = "battles/create_battle.html"
+    form_class = BattleForm
+    model = Battle
+    success_url = reverse_lazy("home")  # TODO: Change success url to the next step
```
[the remark this answers is not in the document]
Comment (pantoja) [reply backend/battles/views.py]: Yes, it's fixed in the next PR that I'll open.

File: .prospector.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (pantoja) [reply .prospector.yaml]: This is in reference to a flag that prospector raised that prevented me from commiting anything on git. Any class based code raised `too-many-ancestors` and `import-error` was in reference to an specific file. I was advised to make this an exception for now.

[the remark this answers is not in the document]
Comment (vanessa) [reply .prospector.yaml]: I understand, but have you tracked this somewhere? Added to notes or something that will make you not forget to remove this exception.

[the remark this answers is not in the document]
Comment (pantoja) [reply .prospector.yaml]: Yes, I'm using the TODO extension on VSCode.

**Pull request merged**